src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,14 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+import React from 'react';
+
+export const CommentInfo = ({ comment }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{comment.name}</strong>
+      {' by '}
+      <a className="CommentInfo__email" href={`mailto:${comment.email}`}>
+        {comment.email}
+      </a>
+    </div>
+    <div className="CommentInfo__body">{comment.body}</div>
+  </div>
```

Comment:
don't forget about code style
```suggestion
    <div className="CommentInfo__title">
      <strong className="CommentInfo__name">
        {comment.name}
      </strong>
      
      {' by '}
      
      <a className="CommentInfo__email" href={`mailto:${comment.email}`}>
        {comment.email}
      </a>
    </div>
    
    <div className="CommentInfo__body">
      {comment.body}
    </div>
  </div>
```